doc/project/release_process.rst

```diff
@@ -453,10 +453,20 @@ steps:
         #. Post a PR with the updated :zephyr_file:`VERSION` file using
            ``release: Zephyr 1.11.0-rc1`` as the commit subject. Merge
            the PR after successful CI.
+
         #. Tag and push the version, using an annotated tag::
 
             $ git pull
             $ git tag -s -m "Zephyr 1.11.0-rc1" v1.11.0-rc1
+
+        #. Verify that the tag has been signed correctly, ``git show`` for the
+           tag must contain a PGP signature (look for the ``BEGIN PGP
```

Comment:
since you've added SSH to this, and I'm not sure what the output looks like yet, should the re-wording also include:
```
for the tag must contain an encrypted key signature
```
as I don't believe the SSH will be a PGP signature.

Reply:
sorry that's right, changed it to "must contain a signature", keep it generic